src/components/common/badge/productBadge.module.css

```diff
@@ -0,0 +1,25 @@
+.productBadge {
+    max-width: 250px;
+    display: flex;
+    visibility: visible;
+    position: absolute;
+    left: 12px;
+    top: 12px;
+    padding: 8px;
+    border-radius: 12px;
+    background-color: var(--white);
+  }
+
+.svgWrapper {
+    padding-right: 8px;
```

Comment:
"padding-right" is incorrect usage for this case, please see what result it outcomes:

![top-sales-badge-review](https://github.com/user-attachments/assets/4d5cd5b1-860f-4cac-840b-f563a98ee158)

Instead, you can use the `margin` property for this element or the `column-gap` property for the parent element.

Reply:
corrected the logo according to your suggestions [ff674c7] https://github.com/boarlabsxyz/Kavoon/pull/60/commits/ff674c7c355dde77665707bb7fb3591d38476224

Reply:
Looks good! 👍